examples/co2_cm1106/co2_cm1106_altitude_comp.ino

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file main.cpp
+ * @date June 2018 - 2021
+ * @brief CO2 cm1106 sensor example
+ * @license GPL3
+ * 
+ * Full documentation:
+ * https://github.com/kike-canaries/canairio_sensorlib#canairio-air-quality-sensors-library
+ * 
+ * Full implementation for WiFi and Bluetooth Air Quality fixed and mobile station:
+ * https://github.com/kike-canaries/canairio_firmware#canairio-firmware
+ * 
+ * CanAirIO project:
+ * https://canair.io
+ */
+
+#include <Arduino.h>
+
+#include <Sensors.hpp>
+
+void onSensorDataOk() {
+    Serial.print(" CO2: " + sensors.getStringCO2());
+    Serial.print(" CO2humi: " + String(sensors.getCO2humi()));
+    Serial.print(" CO2temp: " + String(sensors.getCO2temp()));
+
+    Serial.print(" H: " + String(sensors.getHumidity()));
+    Serial.println(" T: " + String(sensors.getTemperature()));
+     
+}
+
+void onSensorDataError(const char* msg) {
+    Serial.println(msg);
+}
+
+/******************************************************************************
+*  M A I N
+******************************************************************************/
+
+void setup() {
+    Serial.begin(115200);
+    delay(200);
+    Serial.println("\n== Sensor test setup ==\n");
+
+    Serial.println("-->[SETUP] Detecting sensors..");
+
+    sensors.setSampleTime(5);                        // config sensors sample time interval
+    sensors.setOnDataCallBack(&onSensorDataOk);      // all data read callback
+    sensors.setOnErrorCallBack(&onSensorDataError);  // [optional] error callback
+    sensors.setDebugMode(true);
+    sensors.setCO2AltitudeCompensation(2600);        // CO2 altitud comp, example Bogota Colombia
```

Comment:
Maybe in the comment we should write `[optional]` because it is optional parameter.

Reply:
done